### PR TITLE
[Synthetics] Add support for global variables from test

### PR DESCRIPTION
**Pull request by romainberger**

### CODE REVIEW

**Review by sdeprez — DISMISSED** [review]

**Comment by romainberger** [comment]
Please wait before merging, it needs a fix from https://github.com/DataDog/datadog-api-spec/pull/759

**Comment by romainberger** [comment]
Good to merge when approved.

File: datadog/resource_datadog_synthetics_global_variable_test.go [hunk 1]
```diff
@@ -239,15 +254,111 @@ resource "datadog_synthetics_global_variable" "foo" {
...
+       type = "api"
+       subtype = "http"
+
+       request = {
+               method = "GET"
+               url = "https://www.datadoghq.com"
+               timeout = 30
+       }
+
+       assertions = [
```
Comment (therve) [inline datadog/resource_datadog_synthetics_global_variable_test.go:299]: Should we use new assertion format in tests?

File: datadog/resource_datadog_synthetics_global_variable_test.go [hunk 2]
```diff
@@ -239,15 +254,111 @@ resource "datadog_synthetics_global_variable" "foo" {
...
+                       operator = "contains"
+                       target = "application/json"
+               },
+       ]
+
+       locations = [ "aws:eu-central-1" ]
+
+	   options_list {
+	       tick_every = 60
+           follow_redirects = true
```
Comment (therve) [inline datadog/resource_datadog_synthetics_global_variable_test.go:312]: If you can fix indentation that'd be great.

File: datadog/resource_datadog_synthetics_global_variable.go [hunk 3]
```diff
@@ -48,6 +48,47 @@ func resourceDatadogSyntheticsGlobalVariable() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 			},
+			"parse_test_id": {
+				Description: "Id of the Synthetics test to use for a variable from test",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"parse_test_options": {
```
Comment (therve) [inline datadog/resource_datadog_synthetics_global_variable.go:56]: Please add description to all fields defined below.

**Review by therve — CHANGES_REQUESTED** [review]
Some minor comments, otherwise looks good.

**Review by therve — APPROVED** [review]

**Pull request merged**